llvm/test/Transforms/InferAddressSpaces/AMDGPU/select.ll

```diff
@@ -4,10 +4,9 @@
 ;  this doesn't do something insane on non-canonical IR.
 
 ; CHECK-LABEL: @return_select_group_flat(
-; CHECK-NEXT: %cast0 = addrspacecast ptr addrspace(3) %group.ptr.0 to ptr
```

Comment:
I think this also should check the signature of the function; need to make sure you aren't mutating it in a way that breaks uses

Reply:
done